lib/Doctrine/ORM/Query/Exec/FinalizedSelectExecutor.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ORM\Query\Exec;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Result;
+use Doctrine\DBAL\Types\Type;
+
+/**
+ * SQL executor for a given, final, single SELECT SQL query
+ */
+class FinalizedSelectExecutor extends AbstractSqlExecutor
```

Comment:
I'd baseline both issues as well, it's already the case for the parent class: https://github.com/doctrine/orm/blob/82533af893200dd0b28a235d898f12e735c8a76c/psalm-baseline.xml#L1853-L1856